### DISCUSSION TITLE
CSS Grid Content overflow issues when using large gap classes

### DISCUSSION THREAD

**Discussion by doostinharrell** [tailwindlabs/tailwindcss#6045] [Help]
**What version of Tailwind CSS are you using?**
v2.2.19

**What build tool (or framework if it abstracts the build tool) are you using?**
NA

**What version of Node.js are you using?**
NA

**What browser are you using?**
All modern browsers

**What operating system are you using?**
macOS

**Reproduction URL**
https://codepen.io/doostinharrell/pen/PoKBNWB

**Describe your issue**
If you utilize `grid` and a grid gap larger than 20 (ie: `lg:gap-40`) there are content overflow issues when resizing the viewport.

**Comment by adamwathan** [comment]
Hey! Yeah this makes sense and would happen without Tailwind as well, you are trying to add a 160px gap between each column and there are 12 columns so the grid needs at least 1920px to fit on the screen.

Hope that makes sense — the browser is doing exactly what you're asking here.

**Reply by doostinharrell** [reply]
Thanks for clarifying @adamwathan.  I realized this was more of a CSS Grid issue than TailwindCSS after posting the issue.  Converting the grid to 6 columns solves the issue.
